Client/src/Components/Charts/Gauge/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { RadialBarChart, RadialBar, ResponsiveContainer } from "recharts";
+import PropTypes from "prop-types";
+
+const DATA = [{ value: 100 }];
+const PROGRESS_THRESHOLD = 50;
+const DEFAULT_CONTAINER_HEIGHT = 160;
+const DEFAULT_HEADER_FONT = { fontSize: "11px" };
+const DEFAULT_SUBHEADER_FONT = { fontSize: "9px" };
```

Comment:
Is it possible to get this value from our theme?

Reply:
These are values specific to the guage component , I extract them out for readability. like all components can have their own default values

Reply:
https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/94e4d1b1cb9497d56bca5c59a3734595bffb57bd

Reply:
resolved in: https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/94e4d1b1cb9497d56bca5c59a3734595bffb57bd